lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -245,8 +245,8 @@ defmodule Kernel.ParallelCompiler do
     case state.each_cycle.() do
       [] ->
         modules = write_module_binaries(output, beam_timestamp, result)
-        warnings = Enum.reverse(warnings)
-        {:ok, modules, warnings}
+        warnings = Enum.reverse(warnings) ++ check_modules(modules)
+        {:ok, :lists.map(&elem(&1, 0), modules), warnings}
```

Comment:
Any reason for not  using `Enum.map`? I think we should be consistent if we can. Or we always use Enum or we always use `:lists`.